pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsServlet.java

```diff
@@ -100,8 +100,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
                     // this happens when metricsServletTimeoutMs expires
                     // java.lang.IllegalStateException: AsyncContext completed and/or Request lifecycle recycled
                     log.error("Failed to generate prometheus stats, "
-                                    + "this is likely due to metricsServletTimeoutMs: {} ms elapsed",
-                            time, e + "");
+                                    + "this is likely due to metricsServletTimeoutMs: {} ms elapsed", time);
```

Comment:
We should print the toString() representation of the exception, because there would be the case in which it is not the exception that I mentioned in the comment (especially of we upgrade jetty some day and the behaviour changes)

I am sorry for late reply

Reply:
@eolivelli Does you mean 
```
                    log.error("Failed to generate prometheus stats, "
                                    + "this is likely due to metricsServletTimeoutMs: {} ms elapsed {}", time, e+"");
```

Reply:
Yes

Reply:
@eolivelli PTAL, again, thanks

Reply:
@eolivelli  PTAL, again, thanks

Reply:
@eolivelli PTAL, again, thanks

Reply:
@eolivelli PTAL, again, thanks

Reply:
@eolivelli PTAL, again, thanks